README.md

```diff
@@ -139,19 +139,36 @@ This example tells the master the location of the Satellite server (`https://pup
 
 ## Debugging
 
-In addition to looking through the usual reports in the Puppet Enterprise Console, you can also view the Satellite API log file, which may provide clues as to what a particular issue may be. This file is located at `/var/log/httpd/foreman-ssl_access_ssl.log` on your Satellite server.
+In addition to looking through the usual reports in the Puppet Enterprise Console, also review the Satellite API log and the Puppetserver log.
```

Comment:
Are the Satellite API log and Puppet server log also the PE console? If so, maybe just "You can view the Satellite API log and the Puppet server log in the Puppet Enterprise console".

(lowercase for 'console').

Reply:
The Satellite API log and Puppet server log are separate files that must be manually accessed. See lines 144 and 158. Will I just leave this as is then?

Reply:
I'd change this explain you want to look at these files, kind of like the original paragraph. For example: "In addition to the reports in the Puppet Enterprise Console, the Satellite API log and the Puppet server log can help you debug issues".